cmd/kubeadm/app/phases/controlplane/volumes_test.go

```diff
@@ -306,6 +312,10 @@ func TestGetHostPathVolumesForTheControlPlane(t *testing.T) {
 						MountPath: "/etc/kubernetes/controller-manager.conf",
 						ReadOnly:  true,
 					},
+					{
+						Name:      "flexvolume-dir",
+						MountPath: "/usr/libexec/kubernetes/kubelet-plugins/volume/exec",
```

Comment:
Can you please add that `ReadOnly: false` explicitely here instead of relying on the default?

Reply:
Will do